File: concatenation-of-array/concatenation-of-array.cpp
class Solution {
public:
  vector<int> getConcatenation(vector<int>& nums) {
    vector < int > result;

    for ( int i = 0; i < 2; i++ ) {
      for ( auto value : nums ) {
        result.push_back(value);
      }
    }

    return result;
  }
};

File: concatenation-of-array/concatenation-of-array.go
func getConcatenation(nums []int) []int {
    var result []int = []int{}

    for i := 0; i < len(nums); i++ {
        result = append(result, nums[i])
    }

    for i := 0; i < len(nums); i++ {
        result = append(result, nums[i])
    }

    return result
}